packages/core/src/controllers/echo.ts

```diff
@@ -0,0 +1,30 @@
+import { generateChildLogger, Logger } from "@walletconnect/logger";
+import { IEcho } from "@walletconnect/types";
+import { ECHO_CONTEXT, ECHO_URL } from "../constants";
+
+export class Echo extends IEcho {
```

Comment:
nit: Why did we make an entire class for this? Couldn't we have just had a function defined somewhere to register the token? I feel like it's just extra complexity and more state to store needlessly